types/node/util.d.ts

```diff
@@ -1,5 +1,10 @@
 declare module "util" {
     interface InspectOptions extends NodeJS.InspectOptions { }
+    type Style = 'special' | 'number' | 'bigint' | 'boolean' | 'undefined' | 'null' | 'string' | 'symbol' | 'date' | 'regexp' | 'module';
```

Comment:
I think this list is missing `name`, which is used to style the property names:
https://nodejs.org/api/util.html#util_customizing_util_inspect_colors

Reply:
Lol yea https://github.com/nodejs/node/blob/master/lib/internal/util/inspect.js#L389
Feel free to file a PR.